### PR TITLE
Spike Google Analytics 4 Only TurboLinks (Not CSP)

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

**Comment by cazapatamar** [comment]
Please check this implementation and let me know your opinions. CSP is skipped as apparently it is necessary to have SSL certificate.

I tested on GA website and it is created some traffic from my local environment by executing the application.

**Comment by erikasv** [comment]
LGTM but Daniel is who should give the green light.

Also please add the new line at the end of `manifest.js` and  `analytics.js`

**Comment by cazapatamar** [comment]
Deleted the analytics.js.coffee file and this implementation still works

**Comment by danflop** [comment]
@cazapatamar I was checking out the traffic generated by the test and I only see information about the main page. We should test more visiting different sections and also different species using the search bar and also directly using a species URL.

GA4


Production GA

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -2,15 +2,9 @@
...
-			/// Google Analytics//
-			 (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-			 (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-			 m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-			 })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-			 ga('create', 'UA-50977245-1', 'humboldt.org.co');
-			 ga('send', 'pageview');
-			 </script>
+            <!-- Global site tag (gtag.js) - Google Analytics -->
+            <script async src="https://www.googletagmanager.com/gtag/js?id=G-WD07J9FEXQ"></script>
```
Comment (danflop) [inline app/views/layouts/application.html.erb:6]: Please check indentation of this section.

**Review by danflop — CHANGES_REQUESTED** [review]
Check indentation of the changes.

**Comment by cazapatamar** [comment]
> 

@danflop Done. Check again please.

**Review by danflop — APPROVED** [review]

**Pull request merged**